bot/resources/tags/customchecks.md

```diff
@@ -0,0 +1,21 @@
+**Custom Command Checks in discord.py**
+
+You may find yourself in need of a check decorator to do something that doesn't exist in discord.py by default, but fear not, you can make your own! Using discord.py you can use `discord.ext.commands.check` to create you own checks like this:
+```py
+from discord.ext.commands import check, Context
+
+def in_any_channel(*channels):
+  async def predicate(ctx: Context):
+    return ctx.channel.id in channels
+  return check(predicate)
+```
+There's a fair bit to break down here, so let's start with what we're trying to achieve with this check. As you can probably guess from the name it's locking a command to a **list of channels**. The inner function named `predicate` is used to perform the actual check on the command context. Here you can do anything that requires a `Context` object. This inner function should return `True` if the check is **successful** or `False` if the check **fails**.
+
+Here's how we might use our new check:
```

Comment:
Definitely a personal preference, but I really dislike the use of "we" like this. Furthermore, it's inconsistent with the rest of the text, which uses the second-person.